src/func/Microsoft.Health.Fhir.Ingest.Host/Startup.cs

```diff
@@ -0,0 +1,62 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Linq;
+using EnsureThat;
+using Microsoft.ApplicationInsights;
+using Microsoft.ApplicationInsights.Extensibility;
+using Microsoft.Azure.Functions.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Health.Fhir.Ingest.Telemetry;
+
+[assembly: FunctionsStartup(typeof(Microsoft.Health.Fhir.Ingest.Service.Startup))]
```

Comment:
nit: You can leave this here, but I have seen other projects manage assembly attributes by creating a separate AssemblyInfo.cs file. I personally think it is cleaner if assembly attributes are curated in a single location.

Reply:
updated